src/Confluent.Kafka/Consumer.cs

```diff
@@ -734,7 +732,11 @@ public void Stop()
         {
             if (consumerCts == null)
             {
-                throw new Exception("Consumer background poll loop not started - cannot stop.");
+                // Consumer background poll loop not started - cannot stop.
+                // While this might be considered an error there is no point
+                // in throwing an exception here since there are no side-effects
+                // of not being able to stop a non-Started instance.
+                return;
```

Comment:
the refactor in my open PR changes how all this works and thinks all this (stopping on dispose) through carefully. alternatively, we may cut background processing from this PR (I want to think through more whether this actually belongs on consumer or separate, see comments in my open pr on that. there's also the issue of how exceptions on the background thread are surfaced). Anyway, this bit of code is going away regardless.